tests/system/providers/google/cloud/tasks/example_tasks.py

```diff
@@ -48,22 +45,20 @@
 ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
 DAG_ID = "cloud_tasks_tasks"
 
-timestamp = timestamp_pb2.Timestamp()
-timestamp.FromDatetime(datetime.now() + timedelta(hours=12))
+TIMESTAMP = timestamp_pb2.Timestamp()
+TIMESTAMP.FromDatetime(datetime.now() + timedelta(hours=12))
 
-LOCATION = os.environ.get("GCP_APP_ENGINE_LOCATION", "europe-west2")
+REGION = "us-central1"
```

Comment:
May I know why we want to change the variable name from `LOCATION` to `REGION`? Both seem to work. Also, should we keep the env var for easier customization?

Reply:
changed the `region` to `location`,
location is not always referenced in all system tests